_posts/2015-09-30-september-newsletter.md

```diff
@@ -0,0 +1,286 @@
+repetitive. In one of my [latest PRs][event-loop-pr], some libuv "classes" were
+wrapped in a way that makes managing these queues much easier.
+
+### Jemalloc
+
+[jemalloc][jemalloc], a high performance general purpose memory allocator, is
+now used by default. Since Neovim makes heavy use of dynamic queues(see above)
+in its inner loops, `malloc(3)` is called a lot more than Vim, so it is
+important to use a fast implementation that has consistent performance across
+platforms.
+
+In a recent [pr][jemalloc-4-pr], @fmoralesc modified the jemalloc version used
+by our build system to target jemalloc 4.0 which brings even more performance
+enhancements and adds support for more platforms.
+
+### XDG Support
+
+Neovim will support the XDG directory specification. This was
+[proposed][xdg-proposal] by @ZyX-I when the project started, but only a couple
+of months ago we received a [pr][xdg-pr1] from @Yamakaky which was superseded by
+@jck in a [later pr][xdg-pr2].
+
+Following XDG directory specification will let Neovim users to store configuration
+files such as `.nvimrc` and those under `~/.nvim` in the `~/.config` directory,
+which can be overriden by the `$XDG_CONFIG_HOME` environment variable. The
+specification also states that cache files should be stored in a separate
+directory(`~/.local/share`), which is where files like viminfo(now ShaDa) or
+backup/swap can optionally go.
+
+This change makes it simpler for users to backup and manage their configuration
+since it will be stored with other programs that also follow the specification.
+It also keeps the home directory cleaner
+
+### Faster travis builds
+
+We now use [Travis container-based insfrastructure][travis-container] to run
+Neovim builds, which makes CI builds to start immediately. This was
+[implemented][container-pr] by @fwalch, which also did many other improvements
+to our build infrastructure, allowing developers to receive much faster
+feedback when submitting PRs.
+
+### Quickbuild
+
+@jszakmeister is running a [quickbuild][quickbuild] server in his own
+infrastructure. This gives us a backup CI that double checks Neovim PRs, also
+running tests in FreeBSD which is not covered by travis. Thank you for improving
+Neovim robustness @jszakmeister!
+
+## Third-party development
+
+### Neomake
+
+Did you know that there is an alternative to [syntastic][syntastic] that makes
+use of Neovim asynchronous capabilities? [Neomake][neomake] is the best plugin
+for syntatic checking on Neovim: It is extensible like [syntastic][syntastic]
+and the fact that it uses [job-control][job-control] allows it to perform
+checking in background without blocking the user interface. This is very useful
+for compiled languages that are slower to check(typescript, java, .NET).
+
+The migration from [syntastic][syntastic] is also very trivial, great work
+@benekastah!
+
+### FZF
+
+[fzf][fzf] is a command-line fuzzy finder that thanks to its author(@junnegun,
```

Comment:
@tarruda 
1. L216: `junnegun` -> `junegunn`
2. L220: "It really fast" - missing "is"
3. Might also worth mentioning [fzf.vim](http://github.com/junegunn/fzf.vim) project, but it's your call.